modules/mobile/spec/request/military_information_request_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+
+RSpec.describe 'military_information', type: :request do
+  include SchemaMatchers
+  describe 'GET /mobile/v0/military-service-history' do
+    context 'with a user who has a cached iam session' do
+      before { iam_sign_in }
+      let(:inflection_header) { {'X-Key-Inflection' => 'camel'} }
```

Comment:
Figured this out when I updated my user request spec turns out just setting this isn't enough, you need to pass it into the headers as well 😄 

I added a helper method to the `IAMSessionHelper` class:
```ruby
  def iam_headers
    {
      'Authorization' => "Bearer #{access_token}",
      'X-Key-Inflection' => 'camel'
    }
  end
```

Which you then pass into the headers in the request spec call:
``` ruby
get '/mobile/v0/user', headers: iam_headers
```

Reply:
I think once that's in we won't need `set_key_transform :camel_lower` in the serializer.